tenets/codelingo/psr2/lower-case-constant-values/orig.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+function helloWorld() {
+    print "hello world!";
+    $thing = True;
```

Comment:
Just noticed this. Odd that I got no issues found.

Reply:
I just changed the file

Reply:
Oh, nah this file is ignored in .codelingoignore